libs/community/tests/unit_tests/chat_models/test_litellm.py

```diff
@@ -0,0 +1,25 @@
+"""Standard LangChain interface tests"""
+
+from typing import Type
+
+import pytest
+from langchain_core.language_models import BaseChatModel
+from langchain_standard_tests.unit_tests import (
+    ChatModelUnitTests,  # type: ignore[import-not-found]
+)
+
+from langchain_community.chat_models.litellm import ChatLiteLLM
+
+
+class TestLiteLLMStandard(ChatModelUnitTests):
```

Comment:
Can we configure this to skip if `litellm` is not installed by setting `pytest.mark.requires("litellm")`, similar to how it's done here: https://github.com/langchain-ai/langchain/blob/c4e149d4f18319bef6f1d4c409250c4a0ad21dac/libs/community/tests/unit_tests/chat_models/test_anthropic.py#L17